src/index.html

```diff
@@ -7,8 +7,71 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"
+        class="header__logo"
+      >
+        <img src="images/logo.png"
+          alt="header logo"
+          class="logo__img"
+        >
```

Comment:
fix it everywhere
```suggestion
      <a 
        href="#"
        class="header__logo"
      >
        <img 
          src="images/logo.png"
          alt="header logo"
          class="logo__img"
        >
```

Reply:
![image](https://github.com/mate-academy/layout_moyo-header/assets/116989783/3e947349-67b2-44c9-958d-5a0dd62ed154)
I don't know what's wrong with this link, here screenshot